python/vespa/vespa/application.py

```diff
@@ -15,13 +15,15 @@ def __init__(
         url: str,
         port: Optional[int] = None,
         deployment_message: Optional[List[str]] = None,
+        cert: Optional[str] = None,
     ) -> None:
         """
         Establish a connection with a Vespa application.
 
         :param url: URL
         :param port: Port
         :param deployment_message: Message returned by Vespa engine after deployment.
+        :param cert: Certificate and key pair used for mTLS.
```

Comment:
Please indicate the type of the string. Is it a filename, two PEM encoded objects, etc.

Reply:
Fixed.

Reply:
I initially believed it would be possible to send the actual cert and key, of course ... >_<